test/teslamate/vehicles/vehicle/charging_test.exs

```diff
@@ -43,6 +43,7 @@ defmodule TeslaMate.Vehicles.Vehicle.ChargingTest do
                       date: _,
                       charge_energy_added: 0.1,
                       rated_battery_range_km: 1.61,
+                      charge_rate: 0,
```

Comment:
with a charge_rate of 0 no energy would have been added, innit?

Reply:
If plugged into a charger and using more energy to precondition the cabin than the charger can supply, charge rate becomes 0 when charging. This is why I want to track this specifically.

Reply:
Yeah understood, but I do want to introduce a separate test case for this.